src-vue/src/views/PlayView.vue

```diff
@@ -69,6 +82,10 @@ export default defineComponent({
     color: rgb(168, 168, 168);
 }
 
+.fc-stats__container {
+    margin-top: 3px;
```

Comment:
@GeckoEidechse do you think this is enough spacing?

Reply:
Looks fine to me ^^

![image](https://user-images.githubusercontent.com/40122905/218278735-6b685017-9a82-4752-8906-7d0d474adb20.png)